### PR TITLE
Fix handling of system.* reserved collections

### PR DESCRIPTION
**Pull request by francoisferrand**
- Bump arsenal 8.1.141
- Use `getBucketInfos` from arsenal
- Release 1.14.16

Issue: S3UTILS-182

### CODE REVIEW

File: utils/S3UtilsMongoClient.js [hunk 1]
```diff
@@ -838,62 +838,6 @@ class S3UtilsMongoClient extends MongoClientInterface {
         }
     }
 
-    /*
```
Comment (ghost) [inline utils/S3UtilsMongoClient.js:841]: I don't remember why we had a re-implementation of this function, were we doing more logic here? Just making sure we are not deleting something we should not

Comment (francoisferrand) [reply utils/S3UtilsMongoClient.js:841]: it looks like what is described:

> Overwrite the getBucketInfos method to specially handle the cases that bucket collection exists but bucket is not in metastore collection.
> For now, to make the count-items cronjob more robust, we ignore those "bad buckets"

But this is now the behavior in arsenal

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@senx/warp10": "^1.1.2",
     "JSONStream": "^1.3.5",
-    "arsenal": "git+https://github.com/scality/arsenal#8.1.137",
+    "arsenal": "git+https://github.com/scality/arsenal#8.1.141",
```
Comment (benzekrimaha) [inline package.json]: can't we bump to 8.1.142 as it's the latest release ?

**Review by KillianG — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**